src/cmd_argument.rs

```diff
@@ -21,12 +22,30 @@ where
     }
 }
 
+/// Arguments of type [`OsString`] are passed to the child process
+/// as arguments.
+impl CmdArgument for OsString {
+    #[doc(hidden)]
+    fn prepare_config(self, config: &mut Config) {
+        config.arguments.push(self);
+    }
+}
+
+/// Arguments of type [`OsStr`] are passed to the child process
```

Comment:
```suggestion
/// Arguments of type [`&OsStr`] are passed to the child process
```

Reply:
I actually tried this, and for some reason it produces an error when running `cargo doc`:

```
warning: unresolved link to `&OsStr`
  --> src/cmd_argument.rs:40:24
   |
40 | /// Arguments of type [`&OsStr`] are passed to the child process
   |                        ^^^^^^^^ no item named `&OsStr` in scope
   |
   = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default
   = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`

warning: 1 warning emitted
```

It's a bit confusing, since `[&str]` works. Maybe `&str` is special, and otherwise it only works with identifiers and paths, i.e. with no leading `&`?

Reply:
Made this change myself.